assembly.py

```diff
@@ -1682,6 +1681,7 @@ def alignment_summary(inFastaFileOne, inFastaFileTwo, outfileName=None, printCou
     """
     gap = '-'
     ambiguous = 'N'
+    unambiguous = "GATC"
```

Comment:
```suggestion
    unambiguous = set(list("GATCgatc"))
```
Perhaps use `set` for speed's sake, and capture lowercase (just in case) as well?